### ISSUE TITLE
Issue with Set to

### ISSUE THREAD

**Issue by AmberSJones** [ODM2/ODMToolsPython#220]
I have reached some state in this editing session where I cannot perform any edits. As I noted, interpolate (which was previously working) is not functional, and I cannot get 'Set to' to work either. I previously had success with it.

![image](https://cloud.githubusercontent.com/assets/5023186/6985825/907c1196-d9f4-11e4-93c0-711df573b638.png)

**Comment by AmberSJones** [comment]
So, I saved a series and then tried to perform some of these operations. Look at this strange behavior: I selected points and used 'Set to' to set them at 1000. Now the original points are shown as black and are replicated at 1000. Same behavior for interpolate. Same for linear drift correction (see the last screen shot). The points are shown in both black and red. Do we just need to disable editing after a save is performed?

![image](https://cloud.githubusercontent.com/assets/5023186/7507225/e1b61ed6-f429-11e4-8b0d-2fcba484f557.png)
![image](https://cloud.githubusercontent.com/assets/5023186/7507229/f5f664d2-f429-11e4-8ac3-0165bb3445f5.png)
![image](https://cloud.githubusercontent.com/assets/5023186/7507312/3bcd876e-f42b-11e4-9fa7-0248ffd5a3af.png)
